_docs/concepts/traffic-management/manager.md

```diff
@@ -8,12 +8,8 @@ layout: docs
 type: markdown
 ---
 
-Istio-Manager service is responsible for the lifecycle of
-Envoy instances deployed across the Istio service mesh. It exposes
-APIs for [service discovery](https://lyft.github.io/envoy/docs/configuration/cluster_manager/sds_api.html), dynamic updates to [load balancing pools](https://lyft.github.io/envoy/docs/configuration/cluster_manager/cds.html) and
-[routing tables](https://lyft.github.io/envoy/docs/configuration/http_conn_man/rds.html). These API decouple Envoy from
-platform-specific nuances, simplifying the design and increasing
-portability across platforms.
+The Istio-Manager service is responsible for the lifecycle of
```

Comment:
We're treating Istio-Manager as a proper name, so "Istio-Manager is responsible..."